torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -209,6 +214,20 @@ void PyTorchModelLoader::loadConvolution(const torch::jit::Node *ptNode) {
   assert(inputs.size() == 12);
   assert(outputs.size() == 1);
 
+  // Node inputs:
```

Comment:
Then we will have different enum for each operators, wonder if it worth...

Reply:
If we're already going to list out the values in a comment for each operator anyway then IMO there's no real downside, since it'll be the same number of lines of code. And each enum would be contained to the scope of the operator's load function (e.g. `loadConvolution()`).

Reply:
Interesting idea, I had to use old school enum class (class with an anonymous enum in it) so that the values don't leak into the function and I don't need to static_cast everything. Kinda weird but I thinks it's still better.